crates/sui-graphql-rpc/src/types/coin_metadata.rs

```diff
@@ -9,6 +9,7 @@ use async_graphql::*;
 
 use sui_types::coin::CoinMetadata as NativeCoinMetadata;
 
+/// The metadata for an object of type Coin.
 pub(crate) struct CoinMetadata {
     pub super_: MoveObject,
     pub native: NativeCoinMetadata,
```

Comment:
could have a comment similar to Object's native: NativeObject ->     /// Deserialized representation of `stored_object.serialized_object`.